plugins/lmgtfy.py

```diff
@@ -17,8 +17,7 @@
     query = q.strip().replace(" ", seperator)
     reply = url + query
 else:
-    reply = "Discord's best practices for bots state that failed commands should fail silently"
-
-
+    reply = "Discord's best practices for bots state that failed commands " + \
```

Comment:
line limits are a suggestion not a requirement, while you can do this if youre OCD it usually just makes it harder to read